modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/java/com/liferay/dynamic/data/mapping/form/web/internal/display/context/DDMFormDisplayContext.java

```diff
@@ -585,6 +586,34 @@ public boolean isFormShared() {
 		return ParamUtil.getBoolean(_renderRequest, "shared");
 	}
 
+	public boolean isLimitToOneSubmissionPerUser() throws PortalException {
```

Comment:
@marcelabc, I found this name a little confusing. Does the method return a `boolean` indicating if the user has already submitted it?

Reply:
I agree that it is confusing. I will rename 👍